### ISSUE TITLE
Invoice Ninja v5 - Create Invoice node does not persist line items

### ISSUE THREAD

**Issue by parasdaryanani** [n8n-io/n8n#5292]
**Describe the bug**
When creating an invoice with the invoiceninja node, the line items object in the request body is incorrectly formatted. Due to this, line items are blank in new invoices created from n8n. The same happens with Quotes.

**To Reproduce**

1. Add Invoice Ninja node to workflow
2. Set Credentials
3. Set Resource: Invoice
4. Operation: Create
5. Select Client under Additional Fields
6. Create 1 or more Invoice Items (e.g. Cost: 100, Service: X, Hours :2)
7. Execute Node

**Expected Output**

```json
[
  {
    "line_items": [
       {
           "product_key": "X",
           "cost": 100,
           "quantity": 1
       }
    ],
  }
]
```

**Actual Output**

```json
[
  {
    "line_items": [],
  }
]
```

**Environment (please complete the following information):**

- OS: Ubuntu Linux 22.04 (Docker container)
- n8n Version 0.213.0
- Node.js Version 16.17.0
- Database system MySQL
- Operation mode `own`

**Additional context**
The invoiceninja v5 API has been updated since the node was created in n8n. 

Updates required in [invoiceInterface.ts](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/InvoiceNinja/invoiceInterface.ts)

```typescript
export interface IItem {
	cost?: number;
	notes?: string;
	product_key?: string;
	quantity?: number; // previously `qty?: number;` for v4
	tax_rate1?: number;
	tax_rate2?: number;
	tax_name1?: string;
	tax_name2?: string;
}

export interface IInvoice {
	auto_bill?: boolean;
	client_id?: number;
	custom_value1?: number;
	custom_value2?: number;
	email_invoice?: boolean;
	email?: string;
	discount?: number;
	due_date?: string;
	invoice_date?: string;
	line_items?: IItem[]; // previously invoice_items?: IItem[]; for v4
	invoice_number?: string;
	invoice_status_id?: number;
	is_amount_discount?: boolean;
	paid?: number;
	partial?: number;
	partial_due_date?: string;
	po_number?: string;
	private_notes?: string;
	public_notes?: string;
	tax_name1?: string;
	tax_name2?: string;
	tax_rate1?: number;
	tax_rate2?: number;
}

```

Updates required in [InvoiceNinja.node.ts
](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/InvoiceNinja/InvoiceNinja.node.ts) lines 899 - 914

```ts
if (invoceItemsValues) {
	const invoiceItems: IItem[] = [];
	for (const itemValue of invoceItemsValues) {
		const item: IItem = {
			cost: itemValue.cost as number,
			notes: itemValue.description as string,
			product_key: itemValue.service as string,
			quantity: itemValue.hours as number, // previously qty: itemValue.hours as number, for v4
			tax_rate1: itemValue.taxRate1 as number,
			tax_rate2: itemValue.taxRate2 as number,
			tax_name1: itemValue.taxName1 as string,
			tax_name2: itemValue.taxName2 as string,
		};
		invoiceItems.push(item);
	}
	body.line_items = invoiceItems; // body.invoice_items = invoiceItems; for v4
}
```

Similar updates will be required for Quotes.

**Comment by Joffcom** [comment]
Hey @parasdaryanani,

Thanks for reporting this one, I have created an internal ticket to update this at some point which we will be tracking as `NODE-231`

**Comment by parasdaryanani** [comment]
Hi @Joffcom, I'd be happy to submit a PR if that works for you?

**Comment by Joffcom** [comment]
Hey @parasdaryanani,

We are always open to PRs, With this one it will need to support v4 and v5

**Comment by Joffcom** [comment]
@parasdaryanani I see the change and it looks good, I will free up some time to give it a test later in the week.

**Comment by parasdaryanani** [comment]
@Joffcom awesome, thank you.

**Comment by Joffcom** [comment]
Hey @parasdaryanani, 

Just given this a test and it looks good, Going to close this issue as I am going to merge the PR shortly.

**Comment by janober** [comment]
Fix got released with `n8n@0.214.0`
